src/block.cpp

```diff
@@ -25,16 +27,20 @@ void Block::move(int rows, int columns)
 }
 
 
-std::vector<Position> Block::getCellPositions()
+std::vector<Position> Block::getCellPositions() const
```

Comment:
Add a `const` to all methods **that are not supposed to change things**! This increases type safety!